webform_confirm_email.api.php

```diff
@@ -1,26 +1,25 @@
 <?php
+
 /**
  * @file
+ * Document hooks invoked by this module.
  */
 
-
 /**
- * React on an email that was confirmed when the user clicked
- * the confirmation link
- *
- * @param $node
- *   The node object of the webform for which an email was confirmed
+ * React when a confirmation link was clicked.
  *
- * @param $submission
- *   The submission object of the webform submission where the user
- *   just confirmed his/her email address
+ * @param object $node
+ *   The webform node of the the submission.
+ * @param object $submission
+ *   The confirmed webform submission.
+ * @param bool $first_confirmation
+ *   If TRUE this was the first time confirmation link for this submission was
+ *   clicked.
```

Comment:
I had to read this sentence a few times, maybe something along the lines of "TRUE if the submission’s confirmed status was changed" is a little clearer?

Reply:
Ok, tried to improve it. What do you think about the new version?

Reply:
Much better! Not sure it's always true but it describes the intention well ;)